packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -390,8 +399,9 @@ export function getPrismaClient(config: GetPrismaClientConfig) {
         }
 
         this._baseDmmf = new BaseDMMFHelper(config.document)
+        const engineProtocol = getQueryEngineProtocol(config.generator)
 
-        if (this._dataProxy) {
+        if (this._dataProxy && engineProtocol === 'graphql') {
```

Comment:
Why only for `graphql` here?

Reply:
This is the code for loading embedded DMMF schema for edge client, we don't need (and don't have) it if json protocol is used.